ecs.php

```diff
@@ -52,19 +51,14 @@
 // See: https://github.com/bolt/core/issues/2519
 error_reporting(error_reporting() & ~E_NOTICE);
 
-return static function (ECSConfig $ecsConfig): void {
-    $parameters = $ecsConfig->parameters();
-
-    $parameters->set('sets', ['clean-code', 'common', 'php70', 'php71', 'psr12', 'symfony', 'symfony-risky']);
-
-    $parameters->set('paths', [
+return ECSConfig::configure()
+    ->withPaths([
         __DIR__ . '/src',
         __DIR__ . '/ecs.php',
-    ]);
-
-    $parameters->set('cache_directory', 'var/cache/ecs');
-
-    $parameters->set('skip', [
+    ])
+    ->withCache('var/cache/ecs')
+    ->withPreparedSets(psr12: true, common: true, cleanCode: true)
```

Comment:
I'm missing configuration compared to the previous one (symfony for example).

Reply:
The prepared set Symfony, Symfony Risky, PHP 7.0, PHP 7.1 has bee dropped by ECS.